virtualization/windowscontainers/deploy-containers/containerd.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Building the container stack
+description: Learn more about new container building blocks available in Windows.
+keywords: LCOW, linux containers, docker, containers, containerd, cri, runhcs, runc
+author: scooley
+ms.date: 11/17/2018
+ms.topic: article
+ms.prod: windows-containers
+ms.service: windows-containers
+ms.assetid: a0e62b32-0c4c-4dd4-9956-8056e9abd9e5
+---
+
+# Container platform tools in Windows
+
+To make it easier for container management tools to run on a Windows with as little extra integration work as possible, we're introducing a Windows counterpart to runc - [runhcs](https://github.com/Microsoft/hcsshim/tree/master/cmd/runhcs).  We have also worked closely with the containerd project to integrate runhcs into [containerd/containerd](https://github.com/containerd/containerd) and [containerd/cri](https://github.com/containerd/cri).
+
+This article will talk about the Windows and Linux container platform, `runhcs`, HCS (Host Compute Service), and `containerd` on Windows.
+
+## Windows and Linux container platform
+
+In Linux environments, container management tools like Docker are built on another, more granular, set of container tools - [runc](https://github.com/opencontainers/runc) and [containerd](https://containerd.io/).
+
+![Docker architecture on Linux](media/docker-on-linux.png)
+
+`runc` is a Linux command line tool for creating and running containers according to the [OCI container runtime specification](https://github.com/opencontainers/runtime-spec).
+
+`containerd` is a daemon that manages container life cycle from downloading and unpacking the container image through container execution and supervision.
+
+On Windows, we initially took a different approach.  When we started working with Docker to support Windows containers, we built directly on the HCS (Host Compute Service).  [This blog post](https://blogs.technet.microsoft.com/virtualization/2017/01/27/introducing-the-host-compute-service-hcs/) is full of information about why we built the HCS and why we took this approach to containers initially.
+
+![Initial Docker Engine architecture on Windows](media/hcs.png)
```

Comment:
Both images (hcs.png and hcs-with-runhcs.png) contain the headline "Architecture In Windows". Here in the markdown the description "Initial Docker Engine architecture on Windows" and "Future Docker Engine architecture on Windows" makes it more clear. It would be helpful to have this description visible for the reader. 
My suggestion: Either remove the headline from the png and insert markdown headlines in front of each image, or adjust the headlines in the image.